content/grants/proposals/submit.md

```diff
@@ -0,0 +1,68 @@
++++
+title = "Guide to Creating a Grant Proposal"
+weight = 1
+description = "A comprehensive guide to creating a grant proposal."
++++
+
+
+The Urbit project relies on our community to help realize its potential. While the Foundation specifys the direction of 
+development through bounties, we encourage anyone with an idea about how to improve Urbit to submit a grant proposal. This
+guide will describe the different types of proposals, the requested format, and the entire workflow for a successful project.
+
+### Types of Proposals
+
+While we will entertain just about any idea that comes from the community, we are particularly interested in "capital goods":
+projects that extend usability or add tooling to the platform. Recent examples are 
+[Bitcoin wallet](../bitcoin-full-node-provider-and-wallet) integration, [WebRTC](../webrtc-gall-agent-and-external-app)
+implementation, or the [Urbit Large File Sytem](../urbit-lfs-filehosting) project.   
+
+Proposal types include:
+
+ - Core Development - low level network or kernaldevelopment
+ - Application Development - userspace development of specific applications
+ - Developer Tools - Libraries, APIs, unit tests, etc.
+ - Design Work - overall website design work
+ - Documentation - writing guides and website content
+ - Community - organizing events and meetups, staffing the help desk
+
+We have need for lots of different skill sets - not just technical.
+
+
+### Proposal Format
+
+Your proposal should include the following sections:
+
+ - Project name 
+ - Brief description of project
+ - Proposal body including:
+   - Project Overview
+   - Project Scope
+   - Detailed behavior description
+   - Detailed design description
+   - Your experience
+   - Why you are the right person for this project
+ - Milestones:
+   - Any sub-goals of project
+   - Specific deliverables
+   - Expected timeframes
+   - Expected compensation in Stars
+
+### Proposal Submission and Project Workflow
+
+Once you have your proposal in order, submit it using the **[Propose a Project](https://airtable.com/shrCi54rEDxgSZr3z)** form. 
+From this point, your project will go through these stages:
+
+ - **Draft**: You will schedule a meeting with a team member to discuss your proposal in detail. There may be some modifications 
```

Comment:
It may not be clear how one will schedule a meeting with a team member. Probably better to say that we'll reach out to schedule a meeting, which in effect is just the calendly link.